scripts/embedding_server.py

```
@@ -0,0 +1,65 @@
+@app.post("/embed")
+def generate_embedding(request: EmbeddingRequest):
+    if model is None:
+        raise HTTPException(status_code=503, detail="Embedding model is not ready")
+
+    # 1. Java가 보낸 텍스트 꺼내기
+    text_to_embed = request.text
+
+    # 2. 임베딩 계산
+    try:
+        embeddings = model.encode([text_to_embed], batch_size=1, max_length=8192)['dense_vecs']
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)